sql/src/main/java/io/crate/metadata/pgcatalog/PgTypeTable.java

```diff
@@ -41,10 +41,14 @@
 import java.util.Collections;
 import java.util.Map;
 
+import static io.crate.metadata.pgcatalog.OidHash.schemaOid;
+
 public class PgTypeTable extends StaticTableInfo {
 
     public static final RelationName IDENT = new RelationName(PgCatalogSchemaInfo.NAME, "pg_type");
 
+    private static final int TYPE_NAMESPACE_OID = schemaOid(PgCatalogSchemaInfo.NAME);
```

Comment:
Could be an Integer as it would be boxed on Expression creation anyway.

Reply:
good point - thanks